observatory-platform/observatory/platform/bigquery.py

```diff
@@ -783,6 +782,21 @@ def bq_list_tables(project_id: str, dataset_id: str) -> List[str]:
     return table_ids
 
 
+def bq_get_table(full_table_id: str) -> Optional[BQTable]:
```

Comment:
The FQ table id is normally just referred to as the table_id. We differentiate the table names by calling them table_name
```suggestion
def bq_get_table(table_id: str) -> Optional[BQTable]:
    """Get a single Bigqury table object from the Google Bigquery API.
    :param table_id: Fully qualified table id.
    :return: The table obecjt from the Bigqury API."""

    bq_client = bigquery.Client()
    try:
        table = bq_client.get_table(table_id)
        return table
    except NotFound:
        logging.info(f"Table is not found! {table_id}")
        return None
```